src/functional/either.ts

```diff
@@ -226,12 +232,21 @@ class __Either {
     }
 
     static fromNullable<T, R>(value: T, defaultVal: R): Error<R> | Ok<T> {
-        if(value == null) {
+        if (value == null) {
             return Error.of(defaultVal)
         }
 
         return Ok.of(value)
     }
+
+    static fromThrowable<T>(fn: () => T): Error<unknown> | Ok<T> {
```

Comment:
I used `unknown` but was unsure if we could use a type here

Reply:
I think `unknown` is the right call here.